src/screens/Tab/ActivityFeedScreen.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, ReactNode } from 'react';
+import { Layout, Text, Icon } from '@ui-kitten/components';
+import { FlatList, StyleSheet, LayoutAnimation, Platform, UIManager } from 'react-native';
+import { useSelector } from 'react-redux';
+import { LinearGradient } from 'expo-linear-gradient';
+import SwipeableItem from '../../components/SwipeableItem';
+import { NewAssignmentItem } from '../../components/NewAssignments/NewAssignmentItem';
+import { windowWidth } from '../../shared/constants';
+
+interface ActivityFeedProps {
+    route;
+    navigation;
+    onSwipe;
+}
+
+if (Platform.OS === 'android') {
+    UIManager.setLayoutAnimationEnabledExperimental &&
+        UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
TIL [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is also possible on functions too: 
```suggestion
    UIManager.setLayoutAnimationEnabledExperimental?.(true);
```
This is more of a nice-to-have because we'd have [to enable optional chaining](https://stackoverflow.com/questions/52602455/optional-chaining-not-enabled-reactnative/52603755#52603755).

Reply:
o nice, it works and I didn't need to do the whole enable optional chaining thing. That thread was from 2017 haha